### PR TITLE
Sort tags when displaying ; use AND for search results

### LINKED ISSUE #2719: Sort tags when displaying them in Contracts/Addresses/Accounts ; use AND for search results
From Slack:
 ```
tags are currently displayed in order of addition
which means different accounts with the same two tags can see them displayed in different orders
should be sorted prior to display
```

```
tag search seems to be OR rather than AND
i.e. search for `mainnet` `public` and i get accounts which are just `mainnet` showing up
```

### PR DESCRIPTION
**Pull request by ngotchac**
Fixes #2719

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 88eb72468ea6d69253fadb1a444c7b6309a38e1f
  * Your subject line is longer than 72 characters

Guidelines are available at https://github.com/ethcore/parity

---
This message was auto-generated by https://gitcop.com

File: js/src/views/Accounts/List/list.js [hunk 1]
```diff
@@ -130,15 +130,15 @@ export default class List extends Component {
...
-            return searchValues
-              .map(searchValue => value.indexOf(searchValue) >= 0)
-              // `current && truth, true` => use tokens as AND
-              // `current || truth, false` => use tokens as OR
-              .reduce((current, truth) => current || truth, false);
+        return searchValues
+          .map(searchValue => {
+            return values
+              .filter(value => value.indexOf(searchValue) >= 0)
+              .length > 0;
```
Comment (derhuerst) [inline js/src/views/Accounts/List/list.js]: Use `Array.prototype.some` here.

Comment (ngotchac) [inline js/src/views/Accounts/List/list.js]: Ah, right!

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 88eb72468ea6d69253fadb1a444c7b6309a38e1f
  * Your subject line is longer than 72 characters

Guidelines are available at https://github.com/ethcore/parity

---
This message was auto-generated by https://gitcop.com

**Pull request merged**